web/client/selectors/catalog.js

```diff
@@ -19,6 +19,9 @@ const selectedStaticServiceTypeSelector =
     (state) => get(state, `catalog.default.staticServices["${get(state, 'catalog.selectedService')}"].type`, "csw");
 const {projectionSelector} = require('./map');
 
+// Picks configured tile sizes from state, otherwise default is 256
+const tileSizeOptionsSelector = state => get(state, 'catalog.default.tileSizes', [256]);
```

Comment:
Let's make it default
```suggestion
// Picks configured tile sizes from state, otherwise default is [256, 512]
const tileSizeOptionsSelector = state => get(state, 'catalog.default.tileSizes', [256, 512]);
```